sBTC SIP

```diff
@@ -0,0 +1,166 @@
+Overall, the sBTC protocol not only addresses the limitations of the Bitcoin scripting system but also provides a secure and decentralized solution for utilizing Bitcoin in various applications.
+
+## Design
+To achieve these goals, this proposal implements the following features for sBTC:
+- sBTC is a SIP-10 token backed 1-1 by BTC.
+- The sBTC peg wallet is maintained by the set of sBTC signers. These signers are responsible for the security and maintenance of the wallet, ensuring that sBTC is redeemable for BTC.
+- To support deposits and withdrawals in sBTC, Signers must collectively make smart contract calls on Stacks. We require each signer to individually sign any contract call and ensure at least 70% of the signer signatures are present for each contract call to be valid.
+- Bitcoin can be converted into sBTC within 3 Bitcoin blocks. sBTC can be converted into Bitcoin within 6 Bitcoin blocks. This ensures sBTC on and off ramps are faster than other BTC assets on the market.
+- SIP-10 token contact remains consistent across sBTC releases. This provides reliability for users and developers, meaning no adjustment from builders will be needed as the system evolves through subsequent releases.
+
+### Iterating Toward an Open Signer Network
+This SIP describes an iterative approach to sBTC development via a release plan that is simpler to implement and helps accelerate the sBTC release timeline, but does not include the complete feature set described in the original sBTC design documents.
+
+Thus, the rollout for sBTC will include two key phases:
+
+**Bootstrap:** In this phase an initial group of Signers selected through an open community governance process will be responsible for Signing sBTC transactions on the network. The initial group of Signers will be unchanged throughout the Bootstrap phase of the release schedule. During this phase, sBTC will have a distinct signer set (known as the “sBTC Bootstrap Signer”). And sBTC will not explicitly be integrated into the Stacks Proof of Transfer consensus. As a result, this release can activate without a hard fork.
+
+**Signer Rotation:** In this phase the full design of the sBTC system with an open and decentralized Signer set is reached, increasing decentralization after the system is further tested in the bootstrap phase. At this point, sBTC is integrated into the Proof of Transfer consensus and Signers receive rewards for fulfilling sBTC operations. This release does require a hard fork.
+
+#### Differences in the sBTC Bootstrap Phase:
+- sBTC Signer eligibility criteria will be confirmed through community governance vote. The eligibility criteria to become an sBTC signer is described below and will take into account Signer performance and availability. sBTC Signers that meet the criteria are confirmed by the sBTC Working Group. A subsequent release will implement an open and dynamically rotating signer set for each Proof of Transfer cycle, as described in the original sBTC design documents.
```

Comment:
A few things:

* Who is the sBTC Working Group, and why do they have a say in the criteria?  Shouldn't the criteria be set _in the SIP_?

* What "original sBTC design documents?" I have a few protocol sketches for how an sBTC system might work, but they're not public (because they have some pretty bad fundamental flaws).